index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Webpack App</title>
```

Comment:
Can we use this opportunity to choose a better name for the `<title>` of our status page?